### DISCUSSION TITLE
Apply Form - Open question

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#462] [Ideas] [answered]
When you apply for something you get questions to answer. 

![image](https://user-images.githubusercontent.com/64191883/138709206-d30326b6-6948-451e-80ab-69ab0af1c280.png)

Sadly enough I'm very tempted to just answer with ''Yes'' rather than giving names of whom im contact with or how. Maybe formulate the question differently?

**Comment by techsmyth** [comment, marked as answer]
Hi Rosa,

Yes this is actually very easy to update, as it is stored as a configuraiton.

![image](https://user-images.githubusercontent.com/30729240/141795707-5c181d32-c6cc-4645-b4c6-58be89265570.png)

Happy to hear any suggestions regarding the set of questions that get asked.

**Reply by Roseghost** [reply]
I think a more fitting question would be "Through which user,organization or medium have you been acquainted with this community?" 
The user will more likely answer how they got on this application.

**Reply by techsmyth** [reply]
Nice suggestion! I suggest you make a PR on the server repo for the following file: https://github.com/alkem-io/server/blob/develop/src/services/platform/configuration/templates/ux-template.json - up for that?!

**Reply by techsmyth** [reply]
Your PR is approved + merged :)
